components/alert/alert.html

```diff
@@ -0,0 +1,131 @@
+
+          <ul class="components-list">
+            <li class="components-list-item">
+              <a href="../../pages/introduction.html">Introduction</a>
+            </li>
+          </ul>
+        </div>
+
+        <div class="components">
+          <h3 class="components-heading">Components</h3>
+
+          <ul class="components-list">
+            <li class="components-list-item">
+              <a href="../avatar/avatar.html">Avatar</a>
+            </li>
+            <li class="components-list-item">
+              <a href="./alert.html" id="active-page">Alert</a>
+            </li>
+            <li class="components-list-item"><a href="#">Badge</a></li>
+            <li class="components-list-item"><a href="#">Button</a></li>
+            <li class="components-list-item"><a href="#">Cards</a></li>
+            <li class="components-list-item"><a href="#">Image</a></li>
+            <li class="components-list-item"><a href="#">Input</a></li>
+          </ul>
+        </div>
+      </div>
+
+      <!-- MAIN SECTION -->
+
+      <section class="main" id="main-section">
+        <h1 class="main-heading">ALERT</h1>
+        <p>
+          Alerts are short important message used to attract user's attention
+          without interrupting the user's flow.
+        </p>
+
+        <h2>Alert</h2>
+        <p>
+          Alerts can be created using `alert` class followed by the `variant`
+          name.
+        </p>
+
+        <!-- Code Output -->
+        <div class="code-output-container">
+          <div class="alert-display-container">
+            <div class="alert alert-primary">This is Primary Alert!</div>
+            <div class="alert alert-secondary">This is Secondary Alert!</div>
+            <div class="alert alert-success">This is Success Alert!</div>
+            <div class="alert alert-warning">This is Warning Alert!</div>
+            <div class="alert alert-danger">This is Danger Alert!</div>
+          </div>
+        </div>
+
+        <div class="code-example-container">
+          <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%28211%2C211%2C211%2C1%29&t=vscode&wt=none&l=auto&width=872&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=15px&ln=false&fl=1&fm=Hack&fs=14.5px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522alert%2520alert-primary%2522%253EThis%2520is%2520Primary%2520Alert%21%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522alert%2520alert-secondary%2522%253EThis%2520is%2520Secondary%2520Alert%21%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522alert%2520alert-success%2522%253EThis%2520is%2520Success%2520Alert%21%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522alert%2520alert-warning%2522%253EThis%2520is%2520Warning%2520Alert%21%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522alert%2520alert-danger%2522%253EThis%2520is%2520Danger%2520Alert%21%253C%252Fdiv%253E"
+            style="
+              width: 872px;
+              height: 154px;
```

Comment:
Inline styles should be avoided, you can select this iframe in your CSS and then give it the desired width.
Also use of pixels makes it less responsive!

Reply:
Thank you for your suggestions, I will make necessary changes before merging it.